### PR TITLE
Mark did things

### PR DESCRIPTION
**Pull request by marktriggs**
A superbly cohesive pull request here

### CODE REVIEW

File: exporter_app/lib/job_state_storage.rb [hunk 1]
```diff
@@ -1,37 +1,82 @@
...
   end
 
   def last_run_of(job)
-    @store.fetch(job.id, null_job_info)
+    @db.with_connection do |conn|
+      conn.prepare("select * from job_state where job_id = ?", [job.id]) do |statement|
+        rs = statement.execute_query
+        begin
+          if rs.next
+            ret = JobState.from_row(rs)
```
Comment (jambun) [inline exporter_app/lib/job_state_storage.rb]: redundant ret?

File: exporter_app/lib/job_state_storage.rb [hunk 2]
```diff
@@ -1,37 +1,82 @@
...
 
   end
 
   def initialize
-    # FIXME: When this gets created:
-    #  - find anything with a status of 'running'
-    #
-    #  - delete them from the storage?  Seems like we want them to run again if
-    #    we're still within the window...
+    java.lang.Class.for_name("org.sqlite.JDBC")
```
Comment (jambun) [inline exporter_app/lib/job_state_storage.rb]: need this here?

File: exporter_app/lib/job_state_storage.rb (hunk 1 above)
Comment (marktriggs) [inline exporter_app/lib/job_state_storage.rb]: Yep!  Debug leftovers.  Fixed :)

**Comment by jambun** [comment]
💫

**Pull request merged**